logs/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## Fluentd-http
+## Fluentd
+
+### v1.15.2 / 2023-06-20
+
+* [DOWNGRADE] Restoring the image version to 0.0.7 in the Fluentd Helm (Coralogix Plugin) 'values.yaml' file
```

Comment:
It's realted?

Reply:
This was just a cleanup of that changelog file that I noticed. It's not related to the work done in this PR, no.

Reply:
Cool, I'm resolving it